crates/bevy_ui/src/geometry.rs

```diff
@@ -282,15 +282,40 @@ impl Val {
     }
 }
 
+/// All the types that should be able to be used in the [`Val`] enum should implement this trait.
+/// It's implemented for all numebr types by default.
```

Comment:
```suggestion
```
I actually don't think that this line is helpful: users can just read the impl blocks.